api/src/pcapi/routes/pro/offerers.py

```diff
@@ -234,3 +237,30 @@ def get_available_reimbursement_points(
             for reimbursement_point in reimbursement_points
         ],
     )
+
+
+@private_api.route("/offerers/<offerer_id>/stats", methods=["GET"])
+@login_required
+@spectree_serialize(
+    response_model=offerers_serialize.OffererStatsResponseModel,
+    api=blueprint.pro_private_schema,
+)
+def get_offerer_stats_dashboard_url(
+    offerer_id: str,
+) -> offerers_serialize.OffererStatsResponseModel:
+    offerer = offerers_models.Offerer.query.get_or_404(dehumanize(offerer_id))
+    check_user_has_access_to_offerer(current_user, dehumanize(offerer_id))  # type: ignore [arg-type]
+
+    METABASE_SITE_URL = settings.METABASE_SITE_URL
+    METABASE_SECRET_KEY = settings.METABASE_SECRET_KEY
+
+    payload = {
+        "resource": {"dashboard": 438},
+        "params": {"siren": [offerer.siren]},
+        "exp": round(time.time()) + (60 * 10),  # 10 minute expiration
```

Comment:
```suggestion
        "exp": datetime.utcnow() + datetime.timedelta(minutes=10)
```
Je pense que c'est plus lisible comme ça sans avoir besoin de commentaire, tu en penses quoi ?